moz_ingest/io_modules/decoders/moz_ingest/common.lua

```diff
@@ -164,18 +164,22 @@ local hour = floor(os.time() / 3600)
 
 local function get_geo_city(ip_addr)
     local city = UNK_GEO
+    local city_geoname_id = UNK_GEO
```

Comment:
If this is expected to be an integer, it might be problematic if we default to a string value when failing to look it up. We should probably just leave it out if the lookup fails.

Reply:
okay, i'll set it to `nil`, and then lua should consider it to not exist when it's put into `msg.Fields`